.github/workflows/duplicate_issue_detector.yaml

```
@@ -0,0 +1,53 @@
+            - Continue the discussion on the existing one
+
+            If not, feel free to clarify how this issue is different.
+
+          show-footer: true
+
+      # ---- Auto-label ONLY when confidence is very high ----
+      - name: Label as duplicate (high confidence)
+        if: |
+          steps.similarity.outputs.similar-issues-found == 'true' &&
+          steps.similarity.outputs.highest-similarity-score >= '80'
+        uses: actions/github-script@v6
```

actionlint (1.7.10)

[error] the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)